Softeq.NetKit.Components.EventBus.Service/Connection/ServiceBusQueueConnection.cs

```diff
@@ -52,7 +52,7 @@ public async ValueTask DisposeAsync()
 
         private static IQueueClient CreateClient(string connectionString, string queueName)
         {
-            var client = new QueueClient(connectionString, queueName, ReceiveMode.PeekLock, RetryPolicy.Default);
+            var client = new QueueClient(connectionString, queueName, ReceiveMode.ReceiveAndDelete, RetryPolicy.Default);
```

Comment:
Probably it makes sense to make ReceiveMode configurable?

Reply:
Discussed. Changed.